app/config.js

```diff
@@ -201,7 +201,7 @@ tag.listView()
         }),
         nga.field('custom', 'template')
             .label('Upper name')
-            .template('{{ entry.values.name.toUpperCase() }}')
```

Comment:
String template are not understandable from now. Moreover, configuration should be changed in this case: that's Angular templating, not React one.

@fzaninotto, shouldn't we force using a function here to be identical between the two projects?